state/state.go

```diff
@@ -281,6 +281,9 @@ func (state *HelmState) DiffReleases(helm helmexec.Interface, additionalValues [
 				if len(errs) == 0 {
 					if err := helm.DiffRelease(release.Name, normalizeChart(state.BaseChartPath, release.Chart), flags...); err != nil {
 						errs = append(errs, err)
+						// display hint to user until https://github.com/databus23/helm-diff/pull/15 is merged
+						hint := "Please make sure that you have run 'helmfile charts' at least once before running 'helmfile diff'."
```

Comment:
`helmfile sync` rather than `helmfile charts`?

Reply:
`sync` is a superset of `charts`, isn't `charts` the minimum to run?

Reply:
It seems like I'm pretty confused with the naming of `helmfile charts` after all. Because it actually deal with releases, rather than charts.

Anyway, I guess `helmfile charts` is a subset of `sync`, because it [sync releases only](https://github.com/roboll/helmfile/blob/df264e273630c7e7af58721d75ba447bc9097964/main.go#L116). If the user ran `helmfile diff` before `helmfile repos`, the subsequent `helmfile charts` run would just fail due to missing chart deps? Then, I think that we'd better to suggest `helmfile sync` rather than `helmfile charts`. Please correct me if I'm missing something!